README.md

```diff
@@ -188,30 +188,68 @@ Removing network hello_default
 ## Installation
 
 Pre-built binaries are available on [GitHub releases](https://github.com/docker/app/releases) for Windows, Linux and macOS.
+Each tarball contains two binaries:
+- `docker-app-plugin-{linux|macos|windows}` which is docker-app as a [docker cli plugin](https://github.com/docker/cli/issues/1534) 
+- `docker-app-standalone-{linux|macos|windows}` which is docker-app as a standalone utility 
+
+To use `docker-app` plugin, just type `docker app` instead of `docker-app` and all the examples will work the same way:
```

Comment:
There are two missing requirements here:
- A master build of docker/cli
- The `docker-app` binary must have the OS name removed from it (and finish with `.exe` for Windows)

Reply:
Done